kd_approach.md

```diff
@@ -0,0 +1,352 @@
+## Project Overview
```

Comment:
This file really shows you planned how the whole thing will work - really impressed! You kept up with documenting your process for each user story